example_config.yml

```diff
@@ -9,6 +9,9 @@ spotify:
 #sync_playlists:
 #  - spotify_id: 1ABCDEqsABCD6EaABCDa0a
 #    tidal_id: a0b1234-0a1b-012a-abcd-a1b234c5d6d7
+# sync favorites playlist
```

Comment:
I think most users would expect that their liked tracks in Spotify should end up in their liked tracks on Tidal. Asking the user to specify a destination playlist is a bit awkward IMO, so I would rather avoid a change to the config here.